optimum/exporters/openvino/model_patcher.py

```diff
@@ -779,6 +782,7 @@ def __exit__(self, exc_type, exc_value, traceback):
 SUPPORT_SDPA = is_torch_version(">", "2.1.0")
 
 
+# TODO: why
```

Comment:
please add more detailed comment

Reply:
that's exactly what I'm asking for, these methods are patched but there's no comment as to why they're patched